notebook/static/notebook/js/notebook.js

```diff
@@ -2845,6 +2845,57 @@ define([
         }
     };
     
+       
+    Notebook.prototype.save_notebook_as = function() {
```

Comment:
I'd probably put this method into `savewidget.js` because of the symmetry with rename.

Reply:
Will do!